vendor/modules.txt

```diff
@@ -242,6 +243,7 @@ golang.org/x/text/transform
 golang.org/x/text/secure/bidirule
 golang.org/x/text/unicode/bidi
 # golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e
+golang.org/x/tools/cover
```

Comment:
I think we could remove this dép since we don't use it anymore

Reply:
Looks like this is an indirect dependency